runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/FakeWindmillServer.java

```diff
@@ -307,10 +369,16 @@ public boolean commitWorkItem(
           WorkItemCommitRequest request,
           Consumer<Windmill.CommitStatus> onDone) {
         LOG.debug("commitWorkStream::commitWorkItem: {}", request);
+        // Sleep before calling the onDone callback.
+        Uninterruptibles.sleepUninterruptibly(commitWorkSleep.getMillis(), TimeUnit.MILLISECONDS);
```

Comment:
do you need this with injected sleep for commit queue?

Reply:
No, this was just left over from rebasing my commits. Thanks for catching!